plugins/modules/dns_record.py

```diff
@@ -0,0 +1,599 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Infoblox Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: dns_record
+short_description: Manage Record
```

Comment:
```suggestion
short_description: Manage DNS Resource Record
```
similar change below and in results

Reply:
Modified